src/cython_cmake/cmake/UseCython.cmake

```diff
@@ -126,18 +129,14 @@ function(Cython_compile_pyx)
   # Place the cython files in the current binary dir if no path given
   # Can use cmake_path for CMake 3.20+
   if(NOT CYTHON_OUTPUT)
-    # cmake_path(GET INPUT STEM basename)
```

Comment:
This comment were added anticipating the support of a newer minimum required CMake version, we could instead conditionally execute the modern code based on the current cmake version ..